gitlab/service/server/server.go

```diff
@@ -0,0 +1,29 @@
+// Package server contains server for gRPC server service
```

Comment:
@stepanvalyavskiy it seems the license header is missed (BTW, don't you know can we enforce some linter for license checks?)

Reply:
`goheader` that is a part of `golangci-lint` can check headers. Pull requested: #68